OpenAI_API/Completions/CompletionRequest.cs

```diff
@@ -95,6 +95,12 @@ public string Prompt
 		[JsonProperty("logprobs")]
 		public int? Logprobs { get; set; }
 
+		/// <summary>
+		/// Echo back the prompt in addition to the completion
+		/// </summary>
+		[JsonProperty("echo")]
+		public bool? Echo { get; set; } = false;
```

Comment:
This should default to `null`, not `false`, since the API does not require this parameter.  The actual behavior here doesn't really make a difference (since the API uses false by default), but for optional API parameters it's generally better to only send them if the user specifies then rather than sending the default explicitly, as  would happen with the `= false` here.  The idea being that if the OpenAI API were to charge the default value of Echo on their end, it would create an mismatch with the SDK to have this hardcoded.

Reply:
that's a great point. making these changes now, will re-submit soon.